src/ol/render/canvas/replaygroup.js

```diff
@@ -130,20 +204,30 @@ ol.render.canvas.ReplayGroup.prototype.forEachFeatureAtCoordinate = function(
     ol.extent.buffer(hitExtent, resolution * this.renderBuffer_, hitExtent);
```

Comment:
I am wondering if you would also have to take the `hitTolerance` into account when calculating the `hitExtent`.

For example: If you have a point at `0,0` (in pixel coordinates) which has a symbol of 300x300 pixels, you would set the `renderBuffer` to 150px, so that the point can be selected properly. Even when clicking at `150,0` the point could be selected.
But now if you set the `hitTolerance` to 10px, you would expect to get the point if you click at `155,0`. But the `hitExtent` would be `5,-150,305,150`, so the point would not be considered for the hit detection.

Does that make sense? What do you think?